### PR TITLE
[CSL 168] Add browse facets endpoint

### PR DESCRIPTION
**Pull request by crgee1**
- https://docs.constructor.io/rest-api.html#browse-facets
- Not accepting format options in order to not require api token

### CODE REVIEW

File: spec/src/modules/browse.js [hunk 1]
```diff
@@ -706,4 +706,100 @@ describe('ConstructorIO - Browse', () => {
...
+        expect(fetchSpy).to.have.been.called;
+        expect(requestedUrlParams).to.have.property('key');
+        expect(requestedUrlParams).to.have.property('i');
+        expect(requestedUrlParams).to.have.property('s');
+        expect(requestedUrlParams).to.have.property('c').to.equal(clientVersion);
+        done();
+      });
+    });
+
+    it('Should return a response with parameters', (done) => {
```
Comment (esezen) [inline spec/src/modules/browse.js:758]: I think this is a duplicate of the previous one

**Review by esezen — COMMENTED** [review]
One comment about the tests and there is a merge conflict. Rest looks great!

**Review by esezen — APPROVED** [review]
LGTM! 🚀

**Review by sblaurock — APPROVED** [review]
@crgee1 a note I changed the method name to `getBrowseFacets` to be more consistent with the rest of the method naming.

**Pull request merged**